index.html

```diff
@@ -13,18 +13,18 @@
       </ul>
     </div>
     <script>    
-      var ul = document.getElementById('js-list');
+      const ul = document.getElementById('js-list');
       for(var i=1; i<=2; i++) {
-      var li = document.createElement('li');
+      const li = document.createElement('li');
       li.textContent = 'a'+i;
       ul.appendChild(li);
-      var a = document.createElement('a');
+      const a = document.createElement('a');
       a.href= 'a'+i+'.html'
-      li.prepend(a);
-      var img = document.createElement('img');
+      li.insertBefore(a, li.firstChild)
+      const img = document.createElement('img');
       img.src = '/img/bookmark.png';
       a.appendChild(img);
```

Comment:
[IMO]
After adding the text of the a tag earlier, you may now be wondering about the position of the img.

The currently specified appendChild will be added to the end of the specified node, so it will be added just before the closing tag of a.

prepend is added at the beginning of the specified node, so it will be placed before the text of the a tag (i.e. at the beginning).

However, since prepend is not supported by IE, you can use insertAdjacentElement, which we talked about last time, instead of prepend.

Reply:
Yes, I didn't use prepend this time because I received the advice last time. I coded it with insertBefore instead, is that okay?